packages/dcos-integration-test/extra/test_tweeter_demo.py

```diff
@@ -0,0 +1,86 @@
+import os
+import time
+
+import requests
+
+from test_packaging import (
+    _skipif_insufficient_resources
```

Comment:
Importing a private method? You should either:
* Make this method public by removing the leading underscore
* Add this method to the `test_helpers` file. This was a poorly named file as it implies it is itself a test. I'm not a fan of importing anything from tests ( I like tests to be terminal leaves in the dependency tree) but that precedent has been set so I won't be a stinker.

Reply:
Do you anticipate it being a huge pain or be more confusing to just rename the file `helpers.py` or `integration_test_helpers.py`?